app/res/values/strings.xml

```diff
@@ -699,5 +699,7 @@
     <string name="personalid_tooltip_location_success_message">We use location data to ensure secure usage of this application.\n\n<a href="https://www.dimagi.com/terms/latest/privacy/">Click here for more info</a>.</string>
 
     <string name="network_forbidden_error">Access denied. Please contact customer support if the problem persists.</string>
+    <string name="new_network_data_success">New data is available; press here to refresh.</string>
+    <string name="new_network_data_failure">Failed to refresh. You may be viewing outdated data.</string>
```

Comment:
I think these messages are good, but I'd love it if we could get some input from the product team (i.e. Kim, Mary, Ashish, or Brendon) if not already